packages/scene-composer/tests/components/panels/ComponentEditor.spec.tsx

```diff
@@ -47,6 +48,11 @@ describe('ComponentEditor renders correct component', () => {
     );
     expect(wrapper.find(MotionIndicatorComponentEditor).length).toBe(1);
 
+    wrapper = shallow(
```

Comment:
Please refactor these tests to using our more modern standards, we should be actively removing our dependencies on enzyme everytime we go.

Also, since you changed this test, please move it to the appropriate location next to the system it's testing as we deprecate the tests folder. This should simplify all the import statements significantly.